packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2099,6 +2125,10 @@ export const components = ({
             color: odysseyTokens.TypographyColorDisabled,
           }),
 
+          "&.Mui-disabled": {
+            opacity: 1,
+          },
```

Comment:
Can we use that thing you did in a different PR?

`ownerState` to get the disabled state for this classname?

Reply:
@KevinGhadyani-Okta I can't actually. ☹️  This is something that MUI does. I tried overriding it with ownerState but, this `.Mui-disabled` selector was more specific.

Reply:
I think `ownerState` includes props from the component, so it's preferred over adding a global classname.